.envrc

```diff
@@ -0,0 +1,2 @@
+export PRJ_ROOT=$PWD
```

Comment:
Interesting trick! Does `direnv` also use project root as `PWD` even if you open the terminal directly in one of its sub directories?

Reply:
Yes, this is necessary so that relative paths in the `.envrc` have the same result, even if the user jumps into a sub-folder directly.

Reply:
`DIRENV_DIR` already exists and can be queried if you use direnv, there may be no need to set `PRJ_ROOT` then.